[WPT] Add CSS text-shadow import

mdubet opened the pull request:
#### e8878aaeea63348720ffe3540b7ff3cadb401d93
<pre>
[WPT] Add CSS text-shadow import
<a href="https://bugs.webkit.org/show_bug.cgi?id=248176">https://bugs.webkit.org/show_bug.cgi?id=248176</a>

Reviewed by Tim Nguyen.

From WPT c6addf5

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-negcoord-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-negcoord-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-negcoord.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-opacity-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-opacity-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-opacity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/blur-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/blur-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/blur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/color-inherit-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/color-inherit-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/color-inherit.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/color-parserorder-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/color-parserorder-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/color-parserorder.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/currentcolor-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/currentcolor-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/currentcolor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/decorations-multiple-zorder-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/decorations-multiple-zorder-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/decorations-multiple-zorder.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/dynamic-blur-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/dynamic-blur-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/dynamic-blur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/lineoverflow-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/lineoverflow-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/lineoverflow.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/multiple-noblur-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/multiple-noblur-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/multiple-noblur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-1-ref2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-2-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-2-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/padding-decoration-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/padding-decoration-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/padding-decoration.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/text-shadow-computed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/text-shadow-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/text-shadow-invalid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/text-shadow-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/text-shadow-valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/text-shadow-valid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/quirks-decor-noblur-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/quirks-decor-noblur-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/quirks-decor-noblur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/standards-decor-noblur-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/standards-decor-noblur-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/standards-decor-noblur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/textindent-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/textindent-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/textindent.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/w3c-import.log: Added.

Canonical link: <a href="https://commits.webkit.org/256926@main">https://commits.webkit.org/256926@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/dbfd15f067e7fa13df0bdb1f1c01c1bbbeea7dc3

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/97213 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/6479 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/30355 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/106731 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/167002 "Built successfully and passed tests") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/6764 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/35212 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/89607 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/103420 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/102879 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/5086 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/83839 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/32065 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/86936 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/3/builds/88760 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/75003 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/500 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/20236 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/483 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/21687 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/4777 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/5283 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/44182 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/1724 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/41007 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Looks good, thanks, but:
- Please undo all the changes in LayoutTests/imported/w3c/resources/import-expectations.json
- Please add the WPT commit hash in the commit message

mdubet committed:
[WPT] Add CSS text-shadow import
https://bugs.webkit.org/show_bug.cgi?id=248176

Reviewed by Tim Nguyen.

From WPT c6addf5

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-negcoord-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-negcoord-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-negcoord.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-opacity-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-opacity-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-opacity.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/basic.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/blur-expected-mismatch.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/blur-notref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/blur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/color-inherit-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/color-inherit-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/color-inherit.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/color-parserorder-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/color-parserorder-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/color-parserorder.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/currentcolor-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/currentcolor-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/currentcolor.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/decorations-multiple-zorder-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/decorations-multiple-zorder-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/decorations-multiple-zorder.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/dynamic-blur-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/dynamic-blur-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/dynamic-blur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/lineoverflow-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/lineoverflow-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/lineoverflow.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/multiple-noblur-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/multiple-noblur-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/multiple-noblur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-1-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-1-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-1-ref2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-1.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-2-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-2-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/overflow-not-scrollable-2.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/padding-decoration-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/padding-decoration-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/padding-decoration.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/text-shadow-computed-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/text-shadow-computed.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/text-shadow-invalid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/text-shadow-invalid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/text-shadow-valid-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/text-shadow-valid.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/parsing/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/quirks-decor-noblur-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/quirks-decor-noblur-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/quirks-decor-noblur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/standards-decor-noblur-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/standards-decor-noblur-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/standards-decor-noblur.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/textindent-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/textindent-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/textindent.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/text-shadow/w3c-import.log: Added.

Canonical link: https://commits.webkit.org/256926@main
